proto/src/internal/error.rs

```diff
@@ -136,6 +140,52 @@ pub enum OperationError {
     MG0003ServerPhaseInvalidForMigration,
     MG0004DomainLevelInDevelopment,
     MG0005GidConstraintsNotMet,
+    //
+    KP0001KeyProviderNotLoaded,
+    KP0002KeyProviderInvalidClass,
+    KP0003KeyProviderInvalidType,
+    KP0004KeyProviderMissingAttributeName,
+    KP0005KeyProviderDuplicate,
+    KP0006KeyObjectJwtEs256Generation,
+    KP0007KeyProviderDefaultNotAvailable,
+    KP0008KeyObjectMissingUuid,
+    KP0009KeyObjectPrivateToDer,
+    KP0010KeyObjectSignerToVerifier,
+    KP0011KeyObjectMissingClass,
+    KP0012KeyObjectMissingProvider,
+    KP0012KeyProviderNotLoaded,
+    KP0013KeyObjectJwsEs256DerInvalid,
+    KP0014KeyObjectSignerToVerifier,
+    KP0015KeyObjectJwsEs256DerInvalid,
+    KP0016KeyObjectJwsEs256DerInvalid,
+    KP0017KeyProviderNoSuchKey,
+    KP0018KeyProviderNoSuchKey,
```

Comment:
Why's there KP0017 and KP0018 both with the same tail-names?

Reply:
(Also... a few others...?)

I guess this is different parts in the code?

Reply:
Yep, each error has a unique "raise site" so that if the error occurs you can identify precisely where it occured.